filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -144,6 +144,80 @@ function createArticleNode($doc, $pubObject) {
 			$url = PKPString::regexp_replace('#://[^\s]+/index.php#', '://example.com/index.php', $url);
 		}
 		$articleNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'DOIWebsiteLink', $url));
+
+		// Adding Collection on the content 
+		//Collection property="crawler-based"
+		$galleys = $pubObject->getData('galleys');
```

Comment:
`$pubObject` can be an article or galley and they do not have `galleys` data. So here, it should be:
`$galleys = $article->getCurrentPublication()->getData('galleys');`